### ISSUE TITLE
CallbackEntryProvider improvement rendering

### ISSUE THREAD

**Issue by waver55** [stefanuebe/vaadin_fullcalendar#137]
Hi,

i noticed a thing that maybe could be approved on. When you go to the demo (Callback Entry Provider) and reduce your network speed via the debugger you can notice two distinct calls and renderings when you change to the next/previous interval.
I wonder if this could be improved on, either via the _calendar.batchRendering_ function, so it renders only once, when all events got fetched, or maybe even combine the _next_ and the _fetchDates_ calls? I'm not sure tho if any of those two are viable solutions. Whats your opinion on this?

Greetings 
waver

**Comment by stefanuebe** [comment]
Hi, 

I tried to see, what you mean and there seem to be multiple potential "candidates": 

a) There are indeed multiple calls to the server, but only one "fetch" of entries, which is so far ok (the other one is the button click). 

b) The rendering itself seems to be an interesting thing. Indeed, when setting a breakpoint at the code, where the client receives the fetched entries from the server, there is already a set of entries rendered, and it then renders the remaining entries.

But this is nothing, that I can apparently influence. My assumption here is, that the native library simply caches entries from the sibling months, when switching to the next one. Might be a bug, might be a feature. I tried wrapping it with a batchrendering, but that did not change anything (anything else would have suprised me tbh).

Since the fetch from the server only happens once, I would for now define this as a non critical "issue" of the native library, that we have no influence on (until someone else can point me to the spot in our code, where a mistake was made of course :) ) and therefore tag it as wontfix. If you gain some additional knowledge or details, please let me now. Same, if your test may have had additional steps, that I maybe did not do (I simply went to the page and switched months).

**Comment by stefanuebe** [comment]
Since no new infos has been provided, I'll close this issue for now.
